app/view_utils/paypal_helper.rb

```diff
@@ -20,4 +20,8 @@ def user_and_community_ready_for_payments?(user, community)
     PaypalHelper.personal_account_prepared?(user, community) &&
       PaypalHelper.community_ready_for_payments?(community)
   end
+
+  def missing_payment_info?(user, community)
+    community.paypal_enabled && user.listings.present? && !user_and_community_ready_for_payments?(user, community)
```

Comment:
Well, maybe. Don't know if it makes that much difference - but might give a better impression when testing.